packages/file_selector/file_selector_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.1+17
```

Comment:
I think this needs to update the "minor" version. not the bugfix/minimal version. 

Also @AbdeMohlbi let us land this pr then wait to land the others for  3 days.